packages/react/src/components/FxEditor/index.tsx

```diff
@@ -113,13 +105,7 @@ const FxEditor: React.FC = () => {
         // formula.rangeResizeTo = $("#luckysheet-functionbox-cell");
       });
     }
-  }, [
-    context.allowEdit,
-    context.luckysheet_cell_selected_move,
-    context.luckysheet_select_save?.length,
-    refs.globalCache,
-    setContext,
-  ]);
+  }, [context, refs.globalCache, setContext]);
```

Comment:
Can you break `context` into multiple `context.xxxx` that `isAllowEdit` is actually using? We aim to avoid unnecessary renders. It's fine to add `// eslint-disable-next-line react-hooks/exhaustive-deps`